cmd/internal/server/handlers/schema_builder_test.go

```diff
@@ -256,3 +256,30 @@ func TestCanDetectDecimalPrecision(t *testing.T) {
 		})
 	}
 }
+
+//Adding an additional unit test for the tinyint(1) as bool handling since the orginals were not wide enough
+
+func TestGetFivetranDataType_TinyintBooleanHandling(t *testing.T) {
+	tests := []struct {
+		mysqlType             string
+		treatTinyIntAsBoolean bool
+		expected              fivetransdk.DataType
+	}{
+		{"tinyint(1)", true, fivetransdk.DataType_BOOLEAN},
```

Comment:
is there any reason these test cases can't fit as cases into the existing `TestSchema_CanPickRightFivetranType` test rather than defining a new test?